corehq/apps/ota/models.py

```diff
@@ -0,0 +1,68 @@
+from cStringIO import StringIO
+from django.db import models
+
+from casexml.apps.phone.restore import stream_response
+from corehq.blobs import get_blob_db
+from corehq.blobs.atomic import AtomicBlobs
+from corehq.blobs.exceptions import NotFound
+
+import uuidfield
+
+
+class DemoUserRestore(models.Model):
+    uuid = uuidfield.UUIDField(unique=True, auto=True, db_index=True)
```

Comment:
The reason I created this was to avoid setting `CommCareUser.demo_restore_id` as `IntProperty` instead of `StringProperty`. But I think it shouldn't matter anyway, changing it back